modules/oe_webtools_laco_widget/oe_webtools_laco_widget.module

```diff
@@ -16,11 +16,21 @@ function oe_webtools_laco_widget_page_bottom(array &$page_bottom) {
   $logger = \Drupal::logger('oe_webtools_laco_widget');
   $config = \Drupal::config('oe_webtools_laco_widget.settings');
 
+  $cache = CacheableMetadata::createFromRenderArray($page_bottom);
```

Comment:
I moved this code up, so it also works for the "guard" conditions.
Without it, enabling LACO back requires a cache clear.